### ISSUE TITLE
Improve a11y in Cart icon

### ISSUE THREAD

**Issue by YordanSoares** [wpovernight/wp-menu-cart#5]
A WP.org user suggested improving the a11y for the Cart icon ([Accessibility improvements](https://wordpress.org/support/topic/accessibility-improvements-3/)) replacing `<i>` tags wrapper for `<span>` instead.

I did some quick research and found these related reading:
- [A11Y-101.com > Icons and links](https://a11y-101.com/development/icons-and-links)
- [FontAwesome > Accessibility](https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility)

**Comment by alexmigf** [comment]
Curious, I did a test with [Axe](https://addons.mozilla.org/pt-PT/firefox/addon/axe-devtools/) and no errors are showed, maybe because we use an aria attribute already: https://github.com/wpovernight/wp-menu-cart/blob/74b88bb4ec83d6801187b2df05cd8601167e3913/wp-menu-cart.php#L609

**Comment by YordanSoares** [comment]
Indeed, I also saw that, so we met the aria attribute side, but I wonder if replacing the `<i>` tags for `<span>` could improve even more a11y and semantic.

**Comment by alexmigf** [comment]
@YordanSoares see this: https://github.com/FortAwesome/Font-Awesome/issues/16790#issuecomment-690179315

Of course we could replace `<i>` for `<span>` if @Spreeuw agrees with that.

**Comment by alexmigf** [comment]
[Here](https://wordpress.org/support/topic/cart-icon-is-not-showing-but-count-and-total-is-work-fine/?view=all) using `<i>` the cart icon doesn't show, replacing with `<span>` solves the issue.

@Spreeuw do you agree with the replacement?

**Comment by Spreeuw** [comment]
@alexmigf I think that's actually a bad example. The offending style from the theme is this:
```CSS
.header-style10 .header-bottom .container i
```

If that was a style targeting `span` (which is more likely to happen, actually), we'd have the same issue...

**Comment by alexmigf** [comment]
Sorry, I wasn't finding the route cause for that, I see now it was the `opacity` in that style.

**Comment by Spreeuw** [comment]
I'm hesitant to change this for backwards compatibility reasons, and  FontAwesome being an industry leading example still using `<i>` too, I see no reason to switch. Meanwhile, we can prepare our code/CSS to no longer be dependent on the `<i>` element so that if we change it to `<span>` in the future, the transition will be more smooth.
